packages/core/workers/src/Worker.js

```diff
@@ -38,9 +38,10 @@ class Worker extends EventEmitter {
 
     this.child = childProcess.fork(childModule, process.argv, options);
 
-    // Unref the child and IPC channel so that the workers don't prevent the main process from exiting
-    this.child.unref();
-    this.child.channel.unref();
+    // TODO: This was causing odd behavior, need to fix
```

Comment:
@DeMoorJasper Was seeing some odd behavior related to this, still don't totally understand, but I'll try to explain. An error was getting thrown because the deserialization of the Config was losing the configPath, which seemed like it was causing the main process to exit immediately without any indication of anything going wrong. Weird thing is once I fixed the deserialization bug, it still sometimes exits early with these lines uncommented, but builds fine when they are removed. One instance is when either the .parcel-cache or dist folder is not created, the main process will exit before completing the build. When I comment out these lines I don't run into that issue. Not sure why these lines would affect creating the cache and dist folders. Haven't spent much time looking into it, but wanted to give you a heads up?

Is it safe to comment out these lines for now? Could you provide some more context why they were needed?